packages/amplify_storage_s3/lib/src/S3List/S3ListOptions.dart

```diff
@@ -11,9 +11,9 @@ class S3ListOptions extends ListOptions {
 
   @override
   Map<String, dynamic> serializeAsMap() {
-    final Map<String, dynamic> optionsMap = {
-      'accessLevel': describeEnum(accessLevel),
-      'targetIdentityId': targetIdentityId
+    final Map<String, dynamic> optionsMap = <String, dynamic>{
+      'accessLevel' : describeEnum(accessLevel),
```

Comment:
uber nit: Is the space before : needed?

Reply:
good point, that spacing is inconsistent with the other files